From 474a2c352ae88283a645c2e455091e1f6661885b Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Mon, 9 May 2016 05:53:13 -0400
Subject: [PATCH 21/22] do not support spinner

It did not support spinner very well, so drop it.

Upstream-Status: Inappropriate [wrlinux specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 pyanaconda/ui/gui/hubs/progress.glade | 12 ------------
 pyanaconda/ui/gui/hubs/progress.py    |  9 ---------
 2 files changed, 21 deletions(-)

diff --git a/pyanaconda/ui/gui/hubs/progress.glade b/pyanaconda/ui/gui/hubs/progress.glade
index e8f85e8..6793eac 100644
--- a/pyanaconda/ui/gui/hubs/progress.glade
+++ b/pyanaconda/ui/gui/hubs/progress.glade
@@ -68,18 +68,6 @@
                         <property name="can_focus">False</property>
                         <property name="spacing">6</property>
                         <child>
-                          <object class="GtkSpinner" id="progressSpinner">
-                            <property name="visible">True</property>
-                            <property name="can_focus">False</property>
-                            <property name="active">True</property>
-                          </object>
-                          <packing>
-                            <property name="expand">False</property>
-                            <property name="fill">True</property>
-                            <property name="position">0</property>
-                          </packing>
-                        </child>
-                        <child>
                           <object class="GtkLabel" id="progressLabel">
                             <property name="visible">True</property>
                             <property name="can_focus">False</property>
diff --git a/pyanaconda/ui/gui/hubs/progress.py b/pyanaconda/ui/gui/hubs/progress.py
index 7fb4f77..31a5c79 100644
--- a/pyanaconda/ui/gui/hubs/progress.py
+++ b/pyanaconda/ui/gui/hubs/progress.py
@@ -71,8 +71,6 @@ class ProgressHub(Hub):
         if reenable_ransom:
             self._start_ransom_notes()
 
-        self._restart_spinner()
-
         GLib.timeout_add(250, self._update_progress, self._configuration_done)
         threadMgr.add(AnacondaThread(name=THREAD_CONFIGURATION, target=doConfiguration,
                                      args=(self.storage, self.payload, self.data, self.instclass)))
@@ -110,8 +108,6 @@ class ProgressHub(Hub):
                 # we are done, stop the progress indication
                 gtk_call_once(self._progressBar.set_fraction, 1.0)
                 gtk_call_once(self._progressLabel.set_text, _("Complete!"))
-                gtk_call_once(self._spinner.stop)
-                gtk_call_once(self._spinner.hide)
 
                 if callback:
                     callback()
@@ -221,7 +217,6 @@ class ProgressHub(Hub):
         self._progressBar = self.builder.get_object("progressBar")
         self._progressLabel = self.builder.get_object("progressLabel")
         self._progressNotebook = self.builder.get_object("progressNotebook")
-        self._spinner = self.builder.get_object("progressSpinner")
 
         lbl = self.builder.get_object("configurationLabel")
         lbl.set_text(_("%s is now successfully installed, but some configuration still needs to be done.\n"
@@ -289,7 +284,3 @@ class ProgressHub(Hub):
 
         gtk_call_once(self._progressLabel.set_text, message)
 
-    @gtk_action_nowait
-    def _restart_spinner(self):
-        self._spinner.show()
-        self._spinner.start()
-- 
2.8.1

